man/src/image/tag.md

```diff
@@ -9,15 +9,16 @@ entire image name including the optional `TAG` after the ':'.
    present, it may optionally be followed by a port number in the format 
    `:8080`. If not present, the command uses Docker's public registry located at
    `registry-1.docker.io` by default. Name components may contain lowercase 
-   characters, digits and separators. A separator is defined as a period, one or 
+   letters, digits and separators. A separator is defined as a period, one or
    two underscores, or one or more dashes. A name component may not start or end 
```

Comment:
Another thought is that we could sidestep the whole dash vs. hyphen vs. minus debate by just showing the `-` character in the text. No strong feelings.